pgxn/neon/walproposer_pg.c

```diff
@@ -871,6 +911,27 @@ libpqwp_connect_start(char *conninfo)
 	keywords[n] = "dbname";
 	values[n] = conninfo;
 	n++;
+	if (safekeeper_sslcert)
+	{
+		keywords[n] = "sslcert";
```

Comment:
sslcert/sslkey are client certificate options. We already authenticate the client with neon_auth_token, so no need to authenticate it with a client certificate.

We need only these options to enable TLS:
1) `sslmode`=`veritfy-full` - to enable TLS and verify server's certificate
2) `sslrootcert` - to specify trusted root CA certificates
3) `slnegotiation`=`direct` - (optional, supported since PostgreSQL 17)  to disable SSL negotiation step to reduce number of RT to establish the connection

PS/SK will use certificates signed by the same root CA, so it's ok to use the same `sslrootcert` for both PS and SK.

`sslmode` should be different to allow enabling SSL for PS/SK separately.

https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNECT-SSLCERT